### DISCUSSION TITLE
No API keys/signals after restore

### DISCUSSION THREAD

**Discussion by karlfcnsw** [datalust/seq-tickets#1792] [Help]
Hi!
We currently have an old Seq implementation (5.0.2497) on Windows and are wanting to migrate to a newer version and use containers. 

I took a backup of Seq data and created a new windows VM to restore it on, that went well then I went through the process of upgrading Seq on this version, this required a few step updates but I got there in the end and the Windows machine is now running and the API keys and signals are all showing fine. I took a backup of this.

Next I spun up an EC2 instance installed docker and the image and then restored the backup however there is no API Keys/Signals it looks like no restore has taken place. There was a warning during the restore but I didn't think this was likely a problem.

root@07657a5322ac:/# /seqsvr/Seq restore -k <removed> -b /data/immediate-8db20e2b5e11199.seqbac --enable-apps -y
[21:41:49 INF] Opening metastore "/Seq/Documents/metastore.flare"
[21:41:49 INF] Restoring from "/data/immediate-8db20e2b5e11199.seqbac"
[21:41:49 INF] Decrypting content
[21:41:49 INF] Extracting content
[21:41:49 INF] Reading manifest
[21:41:49 INF] Checking manifest
[21:41:49 INF] Clearing out existing documents
[21:41:49 INF] Restoring migration state
[21:41:49 INF] Restoring documents
[21:41:49 INF] Import stage completed with 205 entries
[21:41:49 INF] Restoring "seq.app.email 2.0.74"
[21:41:49 WRN] Could not restore the NuGet package "seq.app.email"; manual reconfiguration may be required
[21:41:49 INF] Closing metastore
Restoration is complete.

Deleted the nuget package and imported again and saw different count of entries but still no api keys etc

Any help/suggestions would be appreciated.

**Comment by nblumhardt** [comment]
Hi @karlfcnsw, thanks for reaching out.

Do you have `/data` mapped to something persistent on the new EC2 install? (Container state is lost on each restart, otherwise.)

Sending a copy of the diagnostic report (Settings > Diagnostics > Download a full diagnostic report) through to `support@` would be the next step, if this doesn't help.

Best regards,
Nick

**Reply by nblumhardt** [reply]
Ah - one more thought - did you restore by overriding the container entrypoint, or via a shell session attached to the running container?

It would need to be the first of those to work property (we normally suggest doing the restore using [init scripts](https://docs.datalust.co/docs/init-scripts).

**Reply by karlfcnsw** [reply]
yeah /data is mapped to /mnt/data on the EC2 instance. The restore was done via a shell session, I'll have a look at doing the restore via init scripts.
